packages/jest-circus/src/legacy_code_todo_rewrite/jest-adapter-init.js

```diff
@@ -88,7 +94,7 @@ const runAndTransformResultsToJestFormat = async ({
         status = 'passed';
         break;
       case 'skip':
-        status = 'skipped';
+        status = 'pending';
```

Comment:
in the new framework i called them `skipped` but jasmine uses `pending` to mark skipped tests.
Because of this all skipped tests were printed with ✔︎